### ISSUE TITLE
Support subgrid positioning

### ISSUE THREAD

**Issue by lazurit11** [Deletescape-Media/Lawnchair#2524]
### Describe the feature

Add support for subgrid positioning of home screen items and fractional resizing of widgets

When enabled, this option would allows icons, widgets, and other items to be placed halfway between a row, column, or both a row and column on the home screen.

### Why would it be useful to add?

This gives more control over the layout when combining icons with multiple widgets of different sizes.

This lets icons be placed closer together.

This allows better tiling for hexagon shaped icons.

### Did the feature exist in Lawnchair Legacy?

No

### Example(s)

<img width="1080" height="642" alt="Image" src="https://github.com/user-attachments/assets/483ffa56-aed8-409a-b66d-20d31c5c96fe" />

**Comment by lazurit11** [comment]
bump

**Comment by Freeplayg** [comment]
Would love this. Currently, I have some widgets that have minimum sizes that are just a little too big, and I'm hoping this would alleviate that.

**Comment by Amorphous404** [comment]
Follow the #2822, for further updates regarding this feature 👍

**Comment by lazurit11** [comment]
> #2822 

Judging by the description of linked issue, this is not what I meant. That issue covers forced widget resize, but does not cover non-integer widget sizes. What I want is to be able to set widget size to i.e. 4 by 1.5 - taller than 1 grid stop, but lower than 2 grid stops.

**Comment by Amorphous404** [comment]
Okay, still if you want you drop you input their, and hopefully get a feedback from the devs as the topic is regarding widget resize, 

But do you know of any launcher that can do this, i can only recall smart launcher 😅

**Comment by lazurit11** [comment]
> But do you know of any launcher that can do this

Nova can do that IIRC

**Comment by SuperDragonXD** [comment]
Well, from what I understand, you're requesting subgrid positioning (for widgets). If I remember correctly, I think there's an issue about that.

**Comment by lazurit11** [comment]
> Well, from what I understand, you're requesting subgrid positioning (for widgets). If I remember correctly, I think there's an issue about that.

You are correct, I just didn't know the right term. But I also don't know if #1893 is applicable to LC12[.1], issue seems pretty old. You can close this if you want.

**Comment by Amorphous404** [comment]
![Screenshot_2022-08-23-21-10-36-38_3822ac0e29cfec8970059bb21d01f05e.jpg](https://user-images.githubusercontent.com/96917972/186201868-14fdaec6-5cc0-438a-aa3d-6b7d58da39f3.jpg)


Yes, these option is available in Nova Launcher

**Comment by lazurit11** [comment]
bump

**Comment by replete** [comment]
The reasons I'm still using Nova Launcher are (in order of usefulness):

- Larger grid size (I use 14x9)
- Disable dock
- widget padding removal
- sub-grid alignment (helpful for fitting widgets into display and maximising the usable space - helpful if you want your homescreen to be a useful dashboard)

Really want to use Lawnchair, definitely is the best option and not far off being a Nova Launcher replacement no-brainer

**Comment by lazurit11** [comment]
bump

**Comment by KTibow** [comment]
bump

**Comment by KTibow** [comment]
bump

**Comment by KTibow** [comment]
bump

**Comment by sppidy** [comment]
Triage:
1. No, This doesnt follow the feature_request.yaml
2. Yes, It very well aligns with the vision.
3. No, but there is a similar feature request #2822 
4. This is a good feature to be implemented and also fits with project vision.
